apps/api/src/reservation/api/http/inventory.controller.ts

```diff
@@ -34,29 +40,34 @@ export class InventoryController {
 
   @Post()
   @UseGuards(WebhookSignatureGuard)
-  async create(@Body() body: any) {
-    if (body['_type'] === 'location') {
+  async create(
+    @Body() body: CreateLocationPayload | CreateDeskPayload | CreateRoomPayload,
```

Comment:
```suggestion
    @Body() body: CreateInventoryItemPayload
```
We can have CreateInventoryItemPayload which will be a discriminated union of specific types.